Software/src/qtsingleapplication/common.pri

```diff
@@ -1,4 +1,4 @@
-exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib
+#exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib

```

Comment:
I can reproduce the problem (though the missing file is not a critical error). From what I can see in the original repo a config.pri was never used and compilation fails if that line is evaluated. I'd say the options are
* remove the line entirely, since it's of no use
* leave it there in case someone needs it sometime. In that case add a comment why it's disabled
I don't mind